src/components/craft/settings/SectionSettings.tsx

```diff
@@ -138,6 +143,8 @@ const SectionSettings: CraftElementSettings<SectionProps> = ({ props, onPropsCha
                   ? 'end'
                   : e.target.value === 'center'
                   ? 'center'
+                  : e.target.value === 'none'
+                  ? ''
```

Comment:
If the requirement is to not set justify-content, can reserver the change.
"none" is not available for justify-content.